### ISSUE TITLE
Subscribing RSS

### ISSUE THREAD

**Issue by Pantyhose-X** [ietf-tools/datatracker#6979]
### Description

Support Subscribing RSS so people can tracker in real time https://datatracker.ietf.org/ 

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by Pantyhose-X** [comment]
> "Read the tracker" is too general - what specifically are you looking for feed information about?

I would like to see information on the latest submissions
> The datatracker has several feeds already, for instance, https://datatracker.ietf.org/feed/rfc.
> 
> Discoverability is difficult, but for the moment, tell me what you are looking for specifically, and I'll let you know if there's already a feed for that.

I don't see the RSS feed icon at https://datatracker.ietf.org/

**Comment by Pantyhose-X** [comment]
> Internet-Drafts

yes

**Comment by rjsparks** [comment]
Paul - please add a feed for recent drafts (that returns the same documents that /doc/recent does) and propose a way to advertise the RSS feeds.

**Comment by richsalz** [comment]
I disagree with https://github.com/ietf-tools/datatracker/issues/6979#issuecomment-2325457637. RSS has a much larger deployed base than ATOM, and to serve our community we should do RSS and then RSS and ATOM.  Please see #8652.

**Comment by beedell-roke** [comment]
@rjsparks, if in scope for this issue, I want to subscribe to updates on [`datatracker.ietf.org/group/jmap/about`](https://datatracker.ietf.org/group/jmap/about/). If not relevant, please confirm so, so that I can request this in a new issue.

**Comment by rjsparks** [comment]
@RokeJulianLockhart What _specific_ changes are you looking for notification for at `datatracker.ietf.org/group/jmap/about`?

**Comment by beedell-roke** [comment]
@rjsparks, currently, documents, conferences, and more miscellaneous history entrants. Essentially, the equivalent to subscribing to the commit changelog, if these were hosted on Git.

**Comment by rjsparks** [comment]
For documents, consider the buttons at the bottom of https://datatracker.ietf.org/group/jmap/documents/.

For meetings, I don't _think_ there's an rss/atom feed yet, but consider subscribing a calendar to future jmap meetings using the buttons at https://datatracker.ietf.org/meeting/upcoming?show=jmap for now.
